### ISSUE TITLE
Chance in helping out.transfering code to esp32 from arduino fanatec Emulator?

### ISSUE THREAD

**Issue by Shane87F3Ng** [David-Keen/Thrustmaster-ESP#1]
Hi David!

Thx for sharing your code for the Thrustmaster. I am currently using an emulator, too, on an arduino base and just want to transfer it to esp32. Are you willing to give some hints and help with that?

Cheers, 
Shane

**Comment by David-Keen** [comment]
Hi,

Would it be ok to discuss this on Discord?

My username is: The Nerd#7934

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: Shane87F3Ng ***@***.***>
Sent: Monday, April 4, 2022 4:28:38 PM
To: David-Keen/Thrustmaster-ESP ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [David-Keen/Thrustmaster-ESP] Chance in helping out.transfering code to esp32 from arduino fanatec Emulator? (Issue #1)


Hi David!

Thx for sharing your code for the Thrustmaster. I am currently using an emulator, too, on an arduino base and just want to transfer it to esp32. Are you willing to give some hints and help with that?

Cheers,
Shane

**Comment by Shane87F3Ng** [comment]
Yeah ofc, check your friend requests :)

**Comment by David-Keen** [comment]
Hey. Have you sent a request? Just checking since I've not received one.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Shane87F3Ng** [comment]
Yes i have, my name is different there, will send another one :)

**Comment by David-Keen** [comment]
Seems like it's not working. I've created a server with an invite link. This expires in 7 days so let me know if you need another.

https://discord.gg/sAaAG29M

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
